src/components/ProgressItem.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react"
+import { useContext } from "react"
+import { RegexContext } from "../store/regex-context"
+import classes from "./ProgressItem.module.css"
+
+import Card from "../UI/Card"
+
+const ProgressItem: React.FC<{ word: string; wordIndex: number }> = React.memo(
+  (props) => {
+    const [hover, setHover] = useState<boolean>(false)
+    const regExContext = useContext(RegexContext)
+    const { guessedRegExArray } = regExContext
+
+    const cardHoverHandler = (
+      e: React.MouseEvent<HTMLHeadingElement, MouseEvent>
+    ) => {
+      setHover(true)
+    }
+    const cardLeaveHandler = (
+      e: React.MouseEvent<HTMLHeadingElement, MouseEvent>
+    ) => {
+      setHover(false)
+    }
+
+    let guessedRegExArrayTransformed: string[] = []
+
+    for (let value of guessedRegExArray) {
+      guessedRegExArrayTransformed = [
+        ...guessedRegExArrayTransformed,
+        value,
```

Comment:
can we use value1, value2, value3?

Reply:
Here I used value (3x times) because this is one regex that I`m saving for the pair of 3 guessed words (and want to display this same regex under each of those three words). We can discuss this in real-time, not sure I`m explaining clearly here